chapters/equations.tex

```diff
@@ -696,6 +696,7 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 The equations of a \lstinline!when!-clause are active during initialization, if and only if they are explicitly enabled with \lstinline!initial()!, and only in one of the two forms \lstinline!when initial() then! or \lstinline!when {$\ldots$, initial(), $\ldots$} then! (and similarly for \lstinline!elsewhen! and algorithms see below).
 In this case, the \lstinline!when!-clause equations remain active during the whole initialization phase.
 In case of a \lstinline!reinit(x, expr)! being active during initialization (due to being inside \lstinline!when initial()!) this is interpreted as adding \lstinline!x = expr! (the \lstinline!reinit!-equation) as an initial equation.
+The \lstinline!reinit! handling applies both if directly inside \lstinline!when!-clause or inside an \lstinline!if!-equation in the \lstinline!when!-clause.
```

Comment:
I think this part is too central to #3001 make this PR qualify as a fix at the moment.  Probably an example would be a better way to address the problem than to state something the sounds like a trivial statement.

Reply:
I added an example of reinit similar to the existing examples.

Reply:
It looks like the example has a point that should be illustrated, but what it shows doesn't seem grounded in normative parts of the text.  For example, it seems to rely heavily on constant evaluation of `steadyState` which is forced by the unbalanced initial `if`-equation.

Are we perhaps missing an `if`-equation branch balancing rule for `reinit` inside `when`-clauses triggered by `initial()`?

Reply:
I now added that both in the normative part and as explanation in the example.

Reply:
I like the new example that I think answers the questions in #3001.  Resolving.